build.sbt

```diff
@@ -3,16 +3,16 @@
   *
   * Unauthorized copying of this file via any medium is strictly prohibited.
   *
-  * Copyright (c) 2019-2020 Snowplow Analytics Ltd. All rights reserved.
+  * Copyright (c) 2019-2021 Snowplow Analytics Ltd. All rights reserved.
   */
 lazy val root = project
   .in(file("."))
   .settings(
     name := "snowplow-micro",
     organization := "com.snowplowanalytics.snowplow",
-    version := "1.1.2",
+    version := "1.2.0-rc1",
```

Comment:
Just pointing this out in case you forget to change it.

As an aside - I am now a big fan of [sbt-dynver](https://github.com/dwijnand/sbt-dynver) for setting the version automatically from the github tag.